tests/partitioninfo.spec.coffee

```diff
@@ -2,6 +2,10 @@ m = require('mochainon')
 Promise = require('bluebird')
 partitioninfo = require('../lib/partitioninfo')
 partition = require('../lib/partition')
+bootRecord = require('../lib/boot-record')
+fs = require 'fs'
```

Comment:
Can you include the parens around `'fs'` and move this up? I usually put relative dependencies, like `../lib/partition` below built-in modules, or NPM modules.